internal/k8s/controller.go

```diff
@@ -1309,3 +1324,36 @@ func (c *AviController) Run(stopCh <-chan struct{}) error {
 
 	return nil
 }
+
+func handleIstioSecret(secret *corev1.Secret) {
+	rootCA := secret.Data["root-cert"]
+	sslKey := secret.Data["key"]
+	sslCert := secret.Data["cert-chain"]
+	modelName := lib.IstioModel
+	newAviModel := nodes.NewAviObjectGraph()
```

Comment:
I think we should do this at the graph layer

Reply:
done